_includes/nn-content.html

```diff
@@ -0,0 +1,158 @@
+            <p class="card-desc">NNG provides TLS 1.2 and ZeroTier transports, offering support for robust and industry
+              standard authentication and encryption. In addition, it is hardened to be resilient against malicious
+              attackers, with special consideration given to use in a hostile Internet.</p>
+          </div>
+        </div>
+        <div class="column is-6">
+          <div class="card feat-card">
+            <img src="/assets/image/feat-usability.png" class="mb-4" />
+            <h3 class="card-title mb-2">Usability</h3>
+            <p class="card-desc">NNG eschews slavish adherence parts of the more complex and less well understood POSIX
+              APIs, while adopting the semantics that are familiar and useful. New APIs are intuitive, and the optional
+              support for separating protocol context and state from sockets makes creating concurrent applications
+              vastly simpler than previously possible.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section id="nn-detail">
+    <div class="container compatibility">
+      <div class="columns">
+        <div class="column is-7">
+          <h2 class="nn-subtitle has-text-left nn-detail-title">Compatibility</h2>
+          <p class="nn-detail-desc">This project offers both wire compatibility and API compatibility, so most nanomsg
+            users can begin using
+            NNG right away.</p>
+          <p class="nn-detail-desc">
+            Existing nanomsg and mangos applications can inter-operate with NNG applications automatically.</p>
+        </div>
+        <div class="column is-4 is-offset-1">
+          <img src="/assets/image/compatibility.png">
+        </div>
+      </div>
+    </div>
+    <div class="container platforms">
+      <div class="columns">
+        <div class="column is-7 is-order-2 is-offset-1">
+          <h2 class="nn-subtitle has-text-right nn-detail-title">Supported Platforms</h2>
+          <p class="nn-detail-desc has-text-right">NNG supports Linux, macOS, Windows (Vista or better), illumos,
+            Solaris, FreeBSD,
+            Android, and iOS. Most other POSIX platforms should work out of the box but have not been tested. Very old
+            versions of otherwise supported platforms might not work.</p>
+        </div>
+        <div class="column is-4">
+          <img src="/assets/image/platforms.png">
+        </div>
+      </div>
+    </div>
+  </section>
+  <section id="nn-quick-start">
+    <div class="container">
+      <h2 class="nn-subtitle has-text-centered nn-quick-start-title has-text-white">
+        Quick Start
+      </h2>
+      <div class="columns is-8-desktop is-variable">
+        <div class="column is-6">
+          <!-- TODO: add a labels' link -->
+          <p class="nn-quick-start-desc mb-3">
+            To build this project, you will need a C99 compatible compiler and <a class="is-brown" href="">CMake</a> version 3.13 or
```

Comment:
The brown color link, on the slate purple or whatever color it is, has pretty poor contrast making it harder to read.  We should pick a color that contrasts better.  Maybe even black, or a more intense form of brown?

Reply:
I agree that I may need to discuss it with my colleagues. Brown is really not very friendly.